kedro-org/kedro-viz
# Fix messaging level when package compatibility is not satisfied

## Description

Resolves #1877 

## Development notes

* Update log level to `warning`
* Modify `PACKAGE_REQUIREMENTS` to have `warning_message` and `min_compatible_version` dict
* Update tests

**NOTE:** Lint will be fixed with the PR - https://github.com/kedro-org/kedro-viz/pull/1903

## QA notes

* Start `kedro viz run` on a kedro project which does not use `kedro-datasets`
* Users should see a warning message instead of error
* All tests should pass

## Checklist
